### PR TITLE
[News] News template update

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
Adds the original link to the template for news.

This PR fixes #

**Notes for Reviewers**
URL is not functional...

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Company/News-single/index.js [hunk 1]
```diff
@@ -56,11 +57,19 @@ const NewsSingle = ({data}) => {
               <MDXRenderer>{body}</MDXRenderer>
             </SRLWrapper>
           </div>
-          <RelatedPosts 
+          <div style={{ display: "flex" }}>
+            <h5>
+              Read the full article on
+            </h5>
+            <a href={frontmatter.eurl} target="_blank" rel="noopener noreferrer">
```
Comment (Chadha93) [inline src/sections/Company/News-single/index.js:64]: @Nikhil-Ladha @debo19 how to reference this eurl to make it functional? I've added it to the query...

**Comment by l5io** [comment]
🚀 Preview for commit a8cfbde5227d6a037e6ef3441248bf55ac8572a5 at: https://629133e72fda411512ba39ec--layer5ng.netlify.app

File: src/sections/Company/News-single/index.js (hunk 1 above)
Comment (debo19) [reply src/sections/Company/News-single/index.js:64]: @Chadha93 add it to the query on `src/templates/news-single.js` as well

**Comment by leecalcote** [comment]
Merge conflict

**Comment by l5io** [comment]
🚀 Preview for commit fe44e8d32abb46d8bb45703ec0bc48c0448c3255 at: https://62913c19c905690057472f29--layer5ng.netlify.app

File: src/sections/Company/News-single/index.js [hunk 2]
```diff
@@ -65,11 +66,19 @@ const NewsSingle = ({data}) => {
               </Col>
             </Row>
           </div>
-          <RelatedPosts 
+          <div style={{ display: "flex" }}>
```
Comment (debo19) [inline src/sections/Company/News-single/index.js:69]: @Chadha93 okay so I think I understood what the problem is. Some news files don't have the `eurl`, mostly if they are our own articles - By`The Newsroom`. Then it makes no sense to add this. Can you add a conditional and display only if the `eurl` is present?

Comment (Chadha93) [reply src/sections/Company/News-single/index.js:69]: Gotcha, now I see, it works with other news having an external link. I'll update it. Thanks. :+1:

**Comment by l5io** [comment]
🚀 Preview for commit cf5b08f4a51072c8632e3f782986118d58b04c5d at: https://62922158bec9cf7c0c4cb1ee--layer5ng.netlify.app

**Review by debo19 — COMMENTED** [review]
Screenshot 2022-05-30 at 1 09 13 PM

Some articles have this already written in their mdx files, that's why it shows as a repetition. We'll have to remove these.

**Comment by Chadha93** [comment]
> Screenshot 2022-05-30 at 1 09 13 PM
> 
> Some articles have this already written in their mdx files, that's why it shows as a repetition. We'll have to remove these.

Hmm, I'll update in this pr only // @debo19

**Comment by l5io** [comment]
🚀 Preview for commit 741a0ea0a39da39b539fdd2a925e78b9e6236e2a at: https://6294b4bbbec9cf20474cb1b4--layer5ng.netlify.app

File: src/sections/Company/News-single/index.js [hunk 3]
```diff
@@ -65,11 +66,23 @@ const NewsSingle = ({data}) => {
...
-          <RelatedPosts 
+          {
+            body && !body.slug && frontmatter.eurl && (
+              <div style={{ display: "flex" }}>
+                <h5>
+                  Read the full article on
+                </h5>&nbsp; 
+                <a href={frontmatter.eurl} target="_blank" rel="noopener noreferrer">
+                  {frontmatter.author}
+                </a>
```
Comment (debo19) [inline src/sections/Company/News-single/index.js:77]:
```suggestion
                </a></h5>
```

File: src/sections/Company/News-single/index.js [hunk 4]
```diff
@@ -65,11 +66,23 @@ const NewsSingle = ({data}) => {
               </Col>
             </Row>
           </div>
-          <RelatedPosts 
+          {
+            body && !body.slug && frontmatter.eurl && (
+              <div style={{ display: "flex" }}>
+                <h5>
+                  Read the full article on
+                </h5>&nbsp;
```
Comment (debo19) [inline src/sections/Company/News-single/index.js:74]:
```suggestion
                
```

File: src/sections/Company/News-single/index.js [hunk 5]
```diff
@@ -65,11 +66,23 @@ const NewsSingle = ({data}) => {
               </Col>
             </Row>
           </div>
-          <RelatedPosts 
+          {
+            body && !body.slug && frontmatter.eurl && (
+              <div style={{ display: "flex" }}>
+                <h5>
+                  Read the full article on
```
Comment (debo19) [inline src/sections/Company/News-single/index.js:73]:
```suggestion
                  Read the full article on 
```

**Review by debo19 — APPROVED** [review]

**Pull request merged**